src/H5win32defs.h

```diff
@@ -52,10 +55,9 @@ struct timezone {
 #define HDgetdrive()         _getdrive()
 #define HDgetlogin()         Wgetlogin()
 #define HDgettimeofday(V, Z) Wgettimeofday(V, Z)
-#define HDisatty(F)          _isatty(F)
+#define isatty(F)            _isatty(F)
```

Comment:
why is this one (and some others)  okay without an HD and others keep HD?
I would prefer not mixing concepts - in one PR.